tools/build.ps1

```diff
@@ -22,10 +22,10 @@ https://openpype.io/docs
 
 #>
 
-$arguments=$ARGS
-$disable_submodule_update=""
-if($arguments -eq "--no-submodule-update") {
-    $disable_submodule_update=$true
+$arguments = $ARGS
```

Comment:
I don't think the changes in this file are intended for this PR?

Reply:
I will merge the latest develop in this PR.